docs/usage.rst

```diff
@@ -42,6 +42,32 @@ and functions like a normal PyVista ``Plotter`` instance. For example:
     sphere.points *= 0.5
 
 
+Multiple Plotters
+~~~~~~~~~~~~~~~~~
+
+The following example shows how to use an interface with multiple plotters. Each
+plotter can be selected and functions like a normal PyVista ``Plotter`` instance:
+
+.. code:: python
+
+    import pyvista as pv
+    from pyvistaqt import MultiPlotter
+
+    mp = MultiPlotter(shape=(2, 2), window_size=(800, 600))
+
+    plotter = mp.select((0, 0))
+    plotter.add_mesh(pv.Sphere())
```

Comment:
```suggestion
    mp[0, 0].add_mesh(pv.Sphere())
```

?

Reply:
Oh this is interesting!

Reply:
I'm sold to use `__getitem__` instead of `select()`. This comes for free and really helps usage.